lib/archethic/bootstrap.ex

```diff
@@ -206,12 +213,91 @@ defmodule Archethic.Bootstrap do
       Logger.info("Synchronization finished")
     end
 
+    persist_gen_addr()
     Sync.publish_end_of_sync()
     SelfRepair.start_scheduler()
 
     :persistent_term.put(:archethic_up, :up)
   end
 
+  # case1: When a node joins a already initialized network
+  # after syncing all transactions persist the gen_addr
+  # case2: node rejoins and requires sync, at end of sync persist gen_addr
+  # case3: first time network initialization
+  def persist_gen_addr() do
```

Comment:
I think that you should not list the transaction here, it will takes a lot of ressources.
As we already knows the seed, you can derive the genesis address directly from the seed.
For exemple, node shared secret would be :
`SharedSecrets.persist_gen_addr()`
-> 
```
def persist_gen_addr() do
    ss_adrr =
      :ets.lookup(@keystore_table, :shared_secrets_index)
      |> Crypto.derive_keypair(0)
      |> elem(0)
      |> Crypto.derive_address()

    origin_software_adrr =
      Archethic.SharedSecrets.get_origin_family_seed(:software)
      |> Crypto.derive_keypair(0)
      |> elem(0)
      |> Crypto.derive_address()

    :persistent_term.put(@ss_gen_key, ss_adrr)
    :persistent_term.put(@origin_gen_key, origin_software_adrr)
  end
```

This can be apply for other chain (exept beacon and oracle chain)